content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -49,6 +45,14 @@ <h3>Summary:</h3>
         </div>
         <br>
```

Comment:
Could we possibly eliminate this `<br>` tag to minimize extra spacing above the note?
```suggestion
```